### PR TITLE
[Diffusion] chore: ensure CFG Zero Star numerical stability for Helios model

### PR DESCRIPTION
**Pull request by RuixiangMa**
## Motivation

Add explicit .float() conversion in Helios's optimized_scale function, align with diffusers [PR #13214](https://github.com/huggingface/diffusers/pull/13214) to ensure CFG Zero Star numerical stability in bf16

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy]
- [ ] (https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review Process

1. Ping Merge Oncalls to start the PR flow. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - `/tag-run-ci-label`, `/rerun-failed-ci`, `/tag-and-rerun-ci`
4. After green CI and required approvals, ask Merge Oncalls to merge.

### CODE REVIEW

File: python/sglang/multimodal_gen/runtime/pipelines_core/stages/model_specific_stages/helios_denoising.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
 def optimized_scale(positive_flat, negative_flat):
     """CFG Zero Star: compute optimal guidance scale."""
+    positive_flat = positive_flat.float()
+    negative_flat = negative_flat.float()
     dot_product = torch.sum(positive_flat * negative_flat, dim=1, keepdim=True)
     squared_norm = torch.sum(negative_flat**2, dim=1, keepdim=True) + 1e-8
     return dot_product / squared_norm
```
[the remark this answers is not in the document]
Comment (RuixiangMa) [reply python/sglang/multimodal_gen/runtime/pipelines_core/stages/model_specific_stages/helios_denoising.py:36]: both approaches are equivalent here，keep the current implementation for clarity.

**Review by mickqian — APPROVED** [review]

**Comment by yhyang201** [comment]
@mickqian Nvidia CI passed and PR is approved, ready for merge

_— SGLDHelper bot_
